app/models/qa/authorities/local/find_works.rb

```diff
@@ -0,0 +1,23 @@
+module Qa::Authorities
```

Comment:
I don't think this belongs in `app/models`, how about `app/authorities` instead? I also don't think we want it in the `Qa` namespace, how about `Sufia`?